### DISCUSSION TITLE
Not seeing my specified Harvest Date

### DISCUSSION THREAD

**Discussion by mahbub-alt** [aquacropos/aquacrop#125] [Q&A]
Hell,
Every time I run my model I specify planting_date = '04/01', harvest_date =  '10/07', but model output is always 09/10 which is one month earlier and I get that same harvest date when I don't specify any Harvest date as well. can anyone tell me how to fix it.

**Comment by Paloschi** [comment]
The behavior you're experiencing with AquaCrop-OSPy setting the harvest date to one month earlier than specified, or defaulting to a specific date when no harvest date is explicitly set, likely stems from how the model interprets crop growth cycles and the parameters defining the end of the growth period.

In AquaCrop-OSPy, the crop's lifecycle—from planting to harvest—is determined by a combination of the crop's genetic characteristics (e.g., length of the growing period, specific growth stages) and environmental factors (e.g., weather conditions). If you're not seeing the expected harvest date, there could be a few reasons:

1. **Automatic Determination of Harvest Date:** AquaCrop-OSPy might automatically determine the harvest date based on the crop's growth and development reaching its conclusion, rather than strictly adhering to the specified harvest date. This can happen if the model's simulation of growth stages completes before the specified harvest date due to the crop's characteristics and the provided environmental conditions.

2. **Model Configuration:** Ensure that the crop type selected and its parameters accurately reflect the expected growth cycle for your conditions. Mismatches here could lead to shorter or longer growth periods than expected.

To address this issue, I sugest you review Crop Parameters, making sure the crop parameters (e.g., Crop object in AquaCrop-OSPy) correctly represent the crop variety you're simulating, as different varieties can have significantly different lengths of growing periods.

Given the unexpected harvest date in your AquaCrop-OSPy model, it's advisable to examine the role of Growing Degree Days (GDD) in determining crop growth and development stages. GDD is a key factor in predicting plant maturation, as it accumulates based on daily maximum and minimum temperatures relative to a crop-specific base temperature. Since each growth stage requires a specific amount of GDD, variations in accumulated GDD due to fluctuating weather conditions could lead to an earlier or later harvest date than initially anticipated. Adjusting the GDD parameters within your crop model, such as the base and upper temperature thresholds, may help align the simulated growth stages more closely with the actual crop lifecycle under your specific environmental conditions.

Cheers
